menu.js

```diff
@@ -0,0 +1,53 @@
+const hamburger = document.querySelector('.nav-menu');
+const navBar = document.querySelector('.navbar');
+const navlist = document.querySelector('.nav-list');
+const portfolio = document.getElementById('portfoliolink');
+const about = document.getElementById('aboutlink');
+const contact = document.getElementById('contactlink');
+
+hamburger.addEventListener('click', () => {
+  navBar.classList.toggle('active');
+  document.querySelector('.nav-bar').classList.toggle('activecolor');
+  hamburger.classList.toggle('cancel');
+  document.querySelector('.headline').classList.toggle('blur');
+  document.querySelector('.work').classList.toggle('blur');
+  document.querySelector('.about').classList.toggle('blur');
+  document.querySelector('.form').classList.toggle('blur');
```

Comment:
- [ ] Please ensure the blur effect is only applied to the mobile menu. This means blur effect even remains sticked when you scroll. To make it more clear, the portifolio sections should be seen through the blur effect with the mobile menu links still seen on the screen. Kindly apply the changes to match the Figma design and have portifolio with better UI ☺️.